app/src/full/java/io/homeassistant/companion/android/thread/ThreadManagerImpl.kt

```diff
@@ -172,7 +173,8 @@ class ThreadManagerImpl @Inject constructor(
                     BORDER_AGENT_ID
                 }
                 ).toByteArray()
```

Comment:
You still need to remove `.toByteArray()` here (and at that point, also remove the outer brackets `()`)